tests/unit-tests/datasets/postfix_formatting/index.rst

```diff
@@ -0,0 +1,8 @@
+:orphan:
```

Comment:
Since this is an index page with a toctree, there is no need to add an `:orphan:` metadata (the same for the child documents that are explicitly referenced inside the `toctree` directive.